test/extensions/common/dynamic_forward_proxy/BUILD

```diff
@@ -19,6 +19,7 @@ envoy_cc_test(
         "//source/extensions/common/dynamic_forward_proxy:dns_cache_manager_impl",
         "//test/mocks/network:network_mocks",
         "//test/mocks/runtime:runtime_mocks",
+        "//test/mocks/server:factory_context_mocks",
```

Comment:
Did you intend for these two BUILD file changes to be in this PR?

Reply:
Yes, it's required because we no longer basically build the world for each test. I think there is an issue with how much test_pch brings in that we need to also fix.